src/providers/twitch/TwitchIrcServer.cpp

```diff
@@ -296,6 +302,31 @@ std::shared_ptr<Channel> TwitchIrcServer::getChannelOrEmptyByID(
     return Channel::getEmpty();
 }
 
+void TwitchIrcServer::bulkRefreshLiveStatus()
+{
+    QStringList userLogins;
+    this->forEachChannel([&userLogins](ChannelPtr chan) {
+        userLogins.push_back(chan->getName());
+    });
```

Comment:
IMO this should use user IDs. The change isn't big, you just have to add a cast to `TwitchChannel` and use `roomId()` instead of `getName()` and update the `Helix::fetchStreams` call. Before this PR Chatterino used user IDs.

Reply:
Make sure to also check if the roomId exists/isn't empty before attempting to push it.

Reply:
Using ids instead of logins now.